builder/tests/builder_test.go

```diff
@@ -70,14 +82,62 @@ func TestBuilder(t *testing.T) {
 			"-v", tmpfile.Name()+":/etc/environment_proxy",
 			imageName)
 	}()
-	dockercli.PrintToStdout(t, stdout, stdoutPipe, "deis-builder running")
+	dockercli.PrintToStdout(t, stdout, stdoutPipe, "Builder is running")
 	if err != nil {
 		t.Fatal(err)
 	}
-	// FIXME: builder needs a few seconds to wake up here!
-	// FIXME: Wait until etcd keys are published
 	time.Sleep(5000 * time.Millisecond)
```

Comment:
The comments are outdated. Code was added earlier to handle those things. So there's nothing left to fix. Or at least, that's how I read the comments.